src/Sulu/Bundle/ContentBundle/Resources/js/stores/WebspaceStore/types.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+export type Webspace = {
+    name: string,
+    key: string,
+    localizations: Array<Localization>,
+    urls: Array<Url>,
+    allLocalizations: Array<LocalizationItem>,
+    portalInformation: Array<PortalInformation>,
+    children: Array<Localization>,
```

Comment:
Doesn't that go into the `Localization`? I think the webspace itself doesn't have any children...